src/main/java/mega/ping/client/PingConfig.java

```diff
@@ -53,6 +53,11 @@ public static class General {
         @Config.LangKey("ping.config.general.deadzone")
         public static double deadzoneRadius;
 
+        @Config.Comment("Enable a beacon beam when using the goto ping")
+        @Config.DefaultBoolean(false)
+        @Config.LangKey("ping.config.general.EnableBeam")
+        public static boolean EnableBeam;
```

Comment:
`EnableBeam` -> `enableBeam`

And add lang into https://github.com/brandyyn/Ping/blob/master/src/main/resources/assets/ping/lang/en_US.lang